src/test/scala/nl.knaw.dans.easy.deposit/DepositDirSpec.scala

```diff
@@ -181,19 +182,17 @@ class DepositDirSpec extends TestSupportFixture with MockFactory {
   "writeSplittedDatasetMetadata" should "write 4 files" in {
     val prologue = """<?xml version='1.0' encoding='UTF-8'?>"""
     val message = "Lorum ipsum"
-    val datasetMetadata = DatasetMetadata(
-      privacySensitiveDataPresent = PrivacySensitiveDataPresent.yes,
-      acceptLicenseAgreement = true,
-      messageForDataManager = Some(message)
-    )
+    val datasetMetadata = DatasetMetadata((File("src") / "test" / "resources" / "manual-test" / "datasetmetadata-from-ui-all.json").contentAsString)
+      .getOrElse(fail("preparing a valid json failed"))
+      .copy(messageForDataManager = Some(message))
     val deposit = createDepositAsPreparation("user001")
     val mdDir = deposit.getDataFiles.getOrElse(fail("preconditions are not met"))
       .filesMetaData.parent.createIfNotExists(asDirectory = true, createParents = true)
     deposit.writeDatasetMetadataJson(datasetMetadata)
     val oldSize = (mdDir / "dataset.json").size
 
-    deposit.splitDatasetMetadata shouldBe Success(())
-    (mdDir / "dataset.json").size should be > oldSize
+    deposit.splitDatasetMetadata(DateTime.now) shouldBe Success(())
+    (mdDir / "dataset.json").size shouldBe oldSize // date submitted is no longer added
```

Comment:
Make this comment

`// while date submitted is added to the dataset metadata, it is not added to the dataset.json file`

The current comment defines the current change, which isn't relevant in the future anymore.